BLL/Helpers/RequestTypeValidator.cs

```diff
@@ -1,15 +1,13 @@
-﻿using System.Collections.Generic;
-using BLL.Models;
-using DAL.Constants;
+﻿using BLL.Models;
 using Microsoft.EntityFrameworkCore.Internal;
 
 namespace BLL.Helpers
 {
     public static class RequestTypeValidator
     {
```

Comment:
Can we use Fluent Validation here instead of custom inplementation?

Reply:
I will use it